src/components/core/Drawer/Drawer.spec.tsx

```diff
@@ -37,8 +37,10 @@ export const drawerState = async <R extends Renderer = ReactRenderer>({
 
     await userEvent.click(closeButton);
 
-    const drawerTitle = screen.getByText("Drawer Title");
+    await sleep(1000);
+
+    const drawerTitle = screen.queryByText("Drawer Title");
 
-    await expect(drawerTitle).not.toBeVisible();
+    await expect(drawerTitle).toBeNull();
```

Comment:
TIL

Reply:
Interestingly, the previous test shouldn't have worked. `findBy` shouldn't have detected anything at all given `lazyMount`, but I guess lazy mounting was improved in Ark v1